packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -241,6 +241,9 @@ function isSafeUse(node: TSESTree.Node): boolean {
     case AST_NODE_TYPES.BinaryExpression:
       return ['instanceof', '==', '!=', '===', '!=='].includes(parent.operator);
 
+    case AST_NODE_TYPES.AssignmentExpression:
+      return parent.operator === '=';
```

Comment:
Could this also check that the node is on the correct side?
i.e.

```ts
class Foo { unbound() {} }
const instance = new Foo();

let x;

x = instance.unbound; // THIS SHOULD ERROR
instance.unbound = x; // THIS SHOULD NOT
```

Reply:
Holy crap! How can you be so considerate!

Reply:
Tests have been updated. It would be great if you could review again.

Reply:
PTAL